scripts/tuple_tree_generator/metaschema.yml

```diff
@@ -0,0 +1,100 @@
+# This file is distributed under the MIT License. See LICENSE.md for details.
+
+"$schema": http://json-schema.org/draft-07/schema#
+"$ref": "#/definitions/Schema"
+definitions:
```

Comment:
Can we enforce the order of entries?

Reply:
You mean the order in which fields are declared? I don't think it's possible to do that, no

Reply:
I didn't know about `propertiesOrder`, json-schema.org does not seem to mention it, neither do a number of python jsonschema validators. Seems it was not implemented. I still added it to the schema, so if one day it will be implemented we'll have this additional guarantee.

Reply:
Uhm, if it's not enforceable maybe it's better to leave it out, what do you say?
If you want to keep it, make sure things are in a sensible order (e.g., inherit is important and should stay on top).

Reply:
On second thought, it does not make a lot of sense to have something we don't enforce, I'm going to drop it